### ISSUE TITLE
Health check not working for SAP HANA (even though app is connected)

### ISSUE THREAD

**Issue by SvendDomdey** [nestjs/terminus#1138]
## I'm submitting a...

<pre><code>
[ ] Regression 
[X] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior
The health check for SAP HANA reports that the database is down while the application is actually connected to SAP HANA. This is quite similar to the issue https://github.com/nestjs/terminus/issues/1102 I raised before.
[We got a requirement to support more databases.]

From the debugging, the error typeorm.health.ts reads as follows:

````
code:257
message:'sql syntax error: line 1 col 8 (at pos 8)'
````

![image](https://user-images.githubusercontent.com/75790355/113008790-a3318780-9177-11eb-9744-a7d21c96f77f.png)

The SQL being sent there is
````
SELECT 1
````

After checking quickly, one option would be to use
````
SELECT now() FROM dummy;
````
instead - that looks like a statement that will run independent of authorizations if connection can be established.
It might be an option to enhance method pingDb in typeorm.health.ts with special logic for SAP HANA:
![image](https://user-images.githubusercontent.com/75790355/113009822-98c3bd80-9178-11eb-9455-ab12738918f5.png)

eg.
`````
case 'sap':
  check = connection.query('SELECT now() FROM dummy;');
  break;
````` 

## Expected behavior
Health check should report database:up when application is connected to it.

## Environment

<pre><code>
@nestjs/typeorm: 7.1.5
@nestjs/terminus: 7.1.1

 
For Tooling issues:
- Node version: XX  
- Platform:  

Others:

</code></pre>

Thanks for your help & kind regards
Svend
PS: I could prepare a PR based on what I  outlined above if that is ok.

**Comment by BrunnerLivio** [comment]
@SvendDomdey hey awesome digging! Yup make sense to me, would be awesome if you can setup a PR! Thanks!

**Comment by SvendDomdey** [comment]
@BrunnerLivio There it is :) Would be great if you can let me know if you have any concerns. Thanks and happy Easter days!

**Comment by BrunnerLivio** [comment]
Released with [7.1.2](https://github.com/nestjs/terminus/releases/tag/7.1.2), thanks a lot for your contribution!
